README.md

```diff
@@ -6,9 +6,9 @@ For more details read the [Community Committee Charter](https://github.com/nodej
 
 ## Why does the Community Committee exist?
 
-The Node.js project has had its fair share of problems regarding diversity, and its fair share of attempts to fix that.
+This Community Committee reflects a formal role and the relevance of the voice of community in the governance of the Node.js project. The formation of this group as a top-level committee alongside the Technical Steering Committee(TSC) is communicated value of the contributors of Node.js that roles other than those of code contributor help foster a healthy, sustainable open source community.
```

Comment:
the second sentence in this is kind of hard to read, and uses complex language. is there a way to make it less confusing for people who don't know what's going on?